src/utils/TodoFilter.tsx

```diff
@@ -0,0 +1,20 @@
+import { TodoStatus } from '../types/TodoStatus';
+import { Todo } from '../types/Todo';
+
+export const visibleTodos = (todos: Todo[], todoStatus: TodoStatus) => {
+  return todos.filter(todo => {
+    switch (todoStatus) {
+      case TodoStatus.ALL:
+        return todo;
+
+      case TodoStatus.ACTIVE:
+        return todo.completed === false;
+
+      case TodoStatus.COMPLETED:
+        return todo.completed === true;
+
```

Comment:
it's better to use `!todo.completed  `and `todo.completed ` here

because they are already Boolean values